src/rt/test/harness.h

```diff
@@ -38,6 +38,12 @@ class SystematicTestHarness
 
   SystematicTestHarness(int argc, char** argv) : opt(argc, argv)
   {
+    for (int i = 0; i < argc; i++)
```

Comment:
If we're going to print the arguments here, then we should remove the [similar printouts](https://github.com/microsoft/verona/blob/1290ba708e9cace2df6b0735377f0e90f07de4d1/src/rt/test/perf/backpressure1/backpressure1.cc#L159) from some of the tests (like the backpressure ones).

Reply:
That particular example doesn't use the harness.  It probably should be rewritten like the runtimepause example, but that can be a different PR after this one.